pkg/cluster/cluster.go

```diff
@@ -333,6 +333,7 @@ func (c *Cluster) startSeedMember(recoverFromBackup bool) error {
 	m := &etcdutil.Member{
 		Name:      etcdutil.CreateMemberName(c.cluster.Metadata.Name, c.memberCounter),
 		Namespace: c.cluster.Metadata.Namespace,
+		Secure:    c.isSecure(),
```

Comment:
well... we already know we might be going to separate them. why not just add a prefix/suffix now for better extensibility. it also improves readability.